gt-profile.yml

```diff
@@ -0,0 +1,115 @@
+BagIt-Profile-Info:
+  BagIt-Profile-Identifier: https://ocr-d.github.io/gt-profile.json
+  BagIt-Profile-Version: '1.2.0'
+  Source-Organization: OCR-D
```

Comment:
What about information about the origin of the digitized lines?
- minimal bibliographic record based on DC?
- and artificially generated lines (+ degeneration)
- what about the degeneration algorithm?

I think that comment may be in the wrong place here. It should probably be placed in this place ## Line metadata##.

Reply:
See https://github.com/OCR-D/spec/pull/105/files/6827085d051e945062203b82ef921e54025cfbda#diff-ee256e83a17cfe309565c88ab376091a That is the definition of what's currently supposed to be in there. Bibliographic metadata would be in the METS referred to by `metsUrl`. How to encode provenance on a line-level I am not sure though. @VolkerHartmann?